src/test/java/emissary/util/FlexibleDateTimeParserTest.java

```diff
@@ -66,14 +66,13 @@ private void test(String date, long expected, DateTimeFormatter formatter) {
     }
 
     /**
-     * Three-letter time zone IDs often point to multiple timezones. Java 8 uses the timezone over the offset causing
-     * problems with the datetime in verifies. Java 9 fixes this issue. Since 9 isn't released and, even if it was, it would
-     * take some time to transition, a regex strips out the short timezone if there is an offset present.
+     * Three-letter time zone IDs often point to multiple timezones. With Java 9+, there are no longer inconsistencies with
```

Comment:
Should we just remove this entire Javadoc comment?

Reply:
I think this whole class is on the chopping block if we get the rest of the updates we want in.